Server/Services/FridgeService.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using EntityTest.Properties;
+using Server.Entities;
+using Server.FridgeResponses;
+using Server.Models;
+
+
+namespace Server.Services
+{
+    public interface IFridgeService
+    {
+        public FridgesResponse GetFridges();
+        public UpdateFridgeResponse GetFridgeById(int fridgeId);
+        public CreateFridgeResponse CreateFridge(CreateFridgeModel createFridgeModel);
+        public UpdateFridgeResponse UpdateFridge(UpdateFridgeModel updateFridgeModel);
+        public DeleteFridgeResponse DeleteFridge(int fridgeId);
+    }
+
+    public class FridgeService : IFridgeService
+    {
+
+        public CreateFridgeResponse CreateFridge(CreateFridgeModel createFridgeModel)
+        {
+            CreateFridgeResponse createFridgeResponse = new CreateFridgeResponse();
+            if (createFridgeModel == null)
+            {
+                createFridgeResponse.StatusResponse = StatusResponse.Error;
+                return createFridgeResponse;
+            }
+
+            if (String.IsNullOrEmpty(createFridgeModel.Description)
+                || String.IsNullOrEmpty(createFridgeModel.Model)
+               )
+            {
+                createFridgeResponse.StatusResponse = StatusResponse.Error;
+                return createFridgeResponse;
+            }
+
+            using (EntityContext db = new EntityContext())
```

Comment:
You should have another one layer with name repository
Where you will provide your context through constructor